scripts/vagrant-linux-priv-buf.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Make sure you grab the latest version
+VERSION=0.30.0
+DOWNLOAD=https://github.com/bufbuild/buf/releases/download/v${VERSION}/buf-Linux-x86_64.tar.gz
```

Comment:
Note for your usage here, you don't need to download the entire tarball, you can just download the binary (drop the `.tar.gz` and you're good to go.

Reply:
I chose the tarball to get `protoc-gen-buf-check-breaking` and `protoc-gen-buf-check-lint`; my understanding was those binaries needed to exist on `$PATH` to do e.g. `buf check lint` in CircleCI in a later commit.

Reply:
Those are not needed for `buf check lint`, those are split out so you can do i.e. `protoc -I . --buf-check-lint_out=.`. All of their functionality is contained within `buf` - what part of the documentation led you to this? We should update it if this was confusing, our apologies.

Reply:
I think it was by going through the [tour](https://docs.buf.build/tour-1), which says:

> This tour assumes you have installed the buf binary on your $PATH. The protoc plugin section assumes you have also installed the protoc-gen-buf-check-breaking and protoc-gen-buf-check-lint binaries, and have a working installation of protoc.

I didn't get as far as https://docs.buf.build/tour-9, which makes it more clear what the "protoc plugin section" means.

My mistake!

Reply:
We'll update the docs to clarify

Reply:
Docs have been updated to not reference the protoc plugin installation until the section of the tour where they are required!

Reply:
Looks great! Thanks!